src/commands/cmd_tdigest.cc

```diff
@@ -180,39 +180,47 @@ class CommandTDigestRevRank : public Commander {
  public:
   Status Parse(const std::vector<std::string> &args) override {
     key_name_ = args[1];
-    inputs_.reserve(args.size() - 2);
-    for (size_t i = 2; i < args.size(); i++) {
-      auto value = ParseFloat(args[i]);
+
+    std::unordered_set<std::string> unique_inputs_set(args.begin() + 2, args.end());
+    origin_inputs_.assign(args.begin() + 2, args.end());
+
+    unique_inputs_.reserve(unique_inputs_set.size());
+    size_t i = 0;
+    for (const auto &input : unique_inputs_set) {
+      auto value = ParseFloat(input);
       if (!value) {
         return {Status::RedisParseErr, errValueIsNotFloat};
       }
-      inputs_.push_back(*value);
+      unique_inputs_.push_back(*value);
+      unique_inputs_order_[input] = i++;
```

Comment:
```suggestion
      unique_inputs_order_[input] = i;
      ++i;
```

Making the increment a newline would make the code better for reading.